amundsen_application/static/js/components/TableDetail/WatermarkLabel/styles.scss

```diff
@@ -8,9 +10,24 @@
     height: 40px;
     margin-right: 8px;
     width: 12px;
+    margin-top: 2px;
   }
 
   .range-labels {
     flex-basis: 42px;
   }
+
+  .date-range {
+    margin-top: 0;
+    margin-bottom: 6px;
```

Comment:
Mind not having this margin or `margin-bottom: 0`

Reply:
That would feel really tight. 
I can bring it down to 4px if you want.

Reply:
I can see that. `4px` would be good.

Reply:
done!